strawberry/schema/base.py

```diff
@@ -20,6 +21,7 @@
 
 class BaseSchema(Protocol):
     config: StrawberryConfig
+    schema_converter: GraphQLCoreConverter
```

Comment:
I don't think this change should be here, but for now it is fine, we'll probably refactor the BaseSchema class at some point :)

Reply:
This is due to this discussion we had in this comment: https://github.com/strawberry-graphql/strawberry/pull/1723#discussion_r825463393

I can remove it if you prefer though :). Or we can keep it for now and refactor `BaseSchema` in the future

Reply:
ah, sorry I didn't review properly last time!